contracts/LinniaRecords.sol

```diff
@@ -185,7 +191,7 @@ contract LinniaRecords is Ownable {
     {
         // validate input
         require(dataHash != 0);
-        require(keccak256(dataUri) != keccak256(""));
+        require(keccak256(abi.encodePacked(dataUri)) != keccak256(abi.encodePacked("")));
         bytes32 metadataHash = keccak256(abi.encodePacked(metadata));
```

Comment:
bytes(metadata)

Reply:
I am following solidity documentation recommendations by using `abi.encodePacked`

Reply:
Where in documentation does it say use encode packed for single argument? Should convert to bytes

Reply:
Here is the warning:
```Warning: This function only accepts a single "bytes" argument. Please use "abi.encodePacked(...)" or a similar function to encode the data.```